src/app/components/file-upload/file-upload.component.ts

```diff
@@ -27,19 +28,60 @@ export class FileUploadComponent implements OnInit {
 
   public regex = /\.([0-9a-z]+)(?=[?#])|(\.)(?:[\w]+)$/gmi;
 
+  public loadTestConfig: string;
+
+  public formData: FormData = new FormData();
+
+  public sessionStorage: Storage;
```

Comment:
I do not believe that `sessionStorage` should be a field of the component.

Reply:
Additionally, any fields that don't need to be accessed within the html through databinding can be labelled as `private`. This does have an impact on the unit tests however. Since the dot notation of accessing fields will not work for private fields in the test files. To get around this, in your test files, you would use bracket notation to access them, such as `myComponent['privateField']`.

Reply:
That is intentional TypeScript behavior for accessing private fields. Ultimately, it will be compiled down to JavaScript, which doesn't have private access anyways.